Core/UserDefaults+NetworkProtection.swift

```diff
@@ -33,7 +33,7 @@ public extension UserDefaults {
 
 public enum NetworkProtectionUserDefaultKeys {
 
-    public static let lastSelectedServer = "com.duckduckgo.network-protection.last-selected-server"
+    public static let lastSelectedServerCity = "com.duckduckgo.network-protection.last-selected-server-city"
```

Comment:
This is now only the city so want to make sure it’s not used for anything expecting more (currently only used for the widget).